app/config/config.yml

```diff
@@ -67,12 +67,17 @@ doctrine:
         types:
             json: Sonata\Doctrine\Types\JsonType
             uuid_binary: Ramsey\Uuid\Doctrine\UuidBinaryType
+        mapping_types:
```

Comment:
add only types that you really need. Add please comment to this config why you use it